lib/meta-box-partial-page.php

```diff
@@ -334,28 +304,29 @@ function the_gutenberg_metaboxes( $locations = array( 'advanced', 'normal', 'sid
 	 * @param array $wp_meta_boxes Global meta box state.
 	 */
 	$wp_meta_boxes = apply_filters( 'filter_gutenberg_meta_boxes', $wp_meta_boxes );
+	$locations     = array( 'side', 'normal', 'advanced' );
 
 	// Render meta boxes.
-	if ( ! empty( $locations ) ) {
-		foreach ( $locations as $location ) {
-			?>
-			<form class="metabox-location-<?php echo $location; ?>">
-				<div id="poststuff" class="sidebar-open">
-					<div id="postbox-container-2" class="postbox-container">
-						<?php
-						gutenberg_meta_box_post_form_hidden_fields( $post, $location );
-						do_meta_boxes(
-							$current_screen,
-							$location,
-							$post
-						);
-						?>
-					</div>
+	?>
+		<form class="metabox-base-form">
+		<?php gutenberg_meta_box_post_form_hidden_fields( $post ); ?>
+		</form>
+		<?php foreach ( $locations as $location ) { ?>
+		<form class="metabox-location-<?php echo esc_attr( $location ); ?>">
+			<div id="poststuff" class="sidebar-open">
+				<div id="postbox-container-2" class="postbox-container">
+					<?php
+					do_meta_boxes(
+						$current_screen,
+						$location,
+						$post
+					);
+					?>
 				</div>
-			</form>
-			<?php
-		}
-	}
+			</div>
+		</form>
+		<?php } ?>
```

Comment:
I don't see a rule for it, but I'd conventionally have expected `if` and `endif` statements for conditions / loops within a PHP template:

>Note that requiring the use of braces just means that single-statement inline control structures are prohibited. You are free to use the alternative syntax for control structures (e.g. if/endif, while/endwhile)—especially in your templates where PHP code is embedded within HTML, for instance:
>
>```php
><?php if ( have_posts() ) : ?>
>    <div class="hfeed">
>        <?php while ( have_posts() ) : the_post(); ?>
>            <article id="post-<?php the_ID() ?>" class="<?php post_class() ?>">
>                <!-- ... -->
>            </article>
>        <?php endwhile; ?>
>    </div>
><?php endif; ?>
>```

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style